src/EditorFeatures/CSharpTest/UseLocalFunction/UseLocalFunctionTests.cs

```diff
@@ -256,6 +256,46 @@ void M()
 }",
 @"using System;
 
+class C
+{
+    void M()
+    {
+        int fibonacci(int v)
+        {
+            if (v <= 1)
+            {
+                return 1;
+            }
+
+            return fibonacci(v - 1, v - 2);
```

Comment:
@CyrusNajmabadi git says you are responsible for this... fibonacci is obviously called with the wrong number number of arguments, is this intentional?

Reply:
not intentional.  but it's doing the right thing.

Reply:
to fix or not to fix?

Reply:
to fix.  the code is technically invalid.  But the fix changes it to be just as invalid as before.  If we didn't fix, users could be confused as to why the fix wasn't be offered.

Reply:
maybe it's good after all to have tests with invalid code?

Reply:
I actually meant whether I should have "fixed" the tests, I agree with should offer a fix even though it's invalid